doc/radosgw/qat-accel.rst

```diff
@@ -69,26 +69,82 @@ hardware is available and QAT is capable to handle it.
 
 Configuration
 =============
-1. QAT based Encryption for RGW 
-
-Edit the Ceph configuration file to make use of QAT based crypto plugin::
+1. Prerequisites
+
+Make sure the QAT driver with version v1.7.L.4.14.0 or higher has
```

Comment:
wrong indent.  see https://ceph--44707.org.readthedocs.build/en/44707/radosgw/qat-accel/#configuration

Reply:
Thanks for reminding. I have made some changes, and you can check it out.